lib/ReactTags.js

```diff
@@ -56,6 +56,42 @@ class ReactTags extends React.Component {
     this.setState({ query })
   }
 
+  handlePaste (e) {
+    // allow over-ride, if there is a need
+    if (this.props.onPaste) {
+      return this.props.onPaste(e)
+    }
+
+    const { delimiterChars } = this.props
+
+    e.preventDefault()
+
+    // get the text data from the clipboard
+    const data = e.clipboardData.getData('Text')
+
+    if (data && delimiterChars.length > 0) {
+      // split the string based on the delimiterChars as a regex
+      const tags = data.split(new RegExp(delimiterChars.join('|'), 'g'))
```

Comment:
If things have to stay regexp, that's an option. Though `delimiterChars` should probably be made sure to contain only single characters, with a warning or error when improper data is passed like `delimiterChars: ['^', '.*\d+\s','lol']`.